Add information on tag format (#21827)
DataDog/documentation: content/en/account_management/billing/aws.md

```
datadog:monitored,env:production,instance-type:c1.*,!region:us-east-1
```

> **Note**: In Datadog, uppercase letters will be changed to lowercase and spaces will be replaced with underscores, for example if you would like to collect metrics from EC2 instances with the tag `Team:Frontend App` in Datadog the tag applied should be `team:frontend_app`.

Reviewer: ```suggestion
**Note**: In Datadog, uppercase letters are changed to lowercase letters and spaces are replaced with underscores. For example, to collect metrics from EC2 instances with the tag `Team:Frontend App`, in Datadog, the tag applied should be `team:frontend_app`.
```

[5]: /infrastructure/
[6]: /help/
[7]: mailto:success@datadoghq.com

> [8]: https://docs.datadoghq.com/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/?tab=cloudformation#streaming-vs-polling

Reviewer: ```suggestion
[8]: /integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/?tab=cloudformation#streaming-vs-polling
```
